pkg/logs/input/docker/tailer.go

```diff
@@ -187,10 +187,12 @@ func (t *Tailer) readForever() {
 				case err == io.EOF:
 					// This error is raised when the container is stopping
 					// or when the container has not started to output logs yet.
```

Comment:
let's update this comment (mention that this can also happen on log file rotation)